README.md

```diff
@@ -15,10 +15,10 @@ supported:
 
 - `rhel-9.6`: RHEL 9.6-based CoreOS; without OpenShift components.
 - `ocp-rhel-9.6`: RHEL 9.6-based CoreOS; including OpenShift components.
-- `c9s`: CentOS Stream-based CoreOS, without OKD components.
-- `okd-c9s`: CentOS Stream-based CoreOS, including OpenShift components. This
-  currently includes some packages from RHEL because not all packages required
-  by OpenShift are provided in CentOS Stream.
+- `c9s`/`c10s`: CentOS Stream-based CoreOS, without OKD components.
+- `okd-c9s`/`okd-c10s`: CentOS Stream-based CoreOS, including OpenShift
```

Comment:
No need to add an `okd-c10s` variant. In fact, let's delete the `okd-c9s` variant.

Reply:
let's delete the okd-c9s variant in another PR

Reply:
https://github.com/openshift/os/pull/1780

Reply:
But still, let's not add `okd-c10s` in this PR.